packages/jest-cli/src/lib/logDebugMessages.js

```diff
@@ -16,7 +16,7 @@ import {version as VERSION} from '../../package.json';
 const logDebugMessages = (
   globalConfig: GlobalConfig,
   config: ProjectConfig,
-  pipe: stream$Writable | tty$WriteStream,
```

Comment:
i wasn't sure what exactly `pipe` object was until i saw the usage of it. I thought `outputStream` would be a more descriptive name